vertx-db2-client/src/main/java/io/vertx/db2client/impl/drda/DRDAQueryResponse.java

```diff
@@ -116,15 +116,50 @@ public void readOpenQuery() { // @AGG removed callback StatementCallbackInterfac
         endOfSameIdChainData();
     }
     
+    /**
+     * Parse the reply for the Open Query Command. This method handles the
+     * parsing of all command replies and reply data for the opnqry command.
+     */
+    public void readBeginOpenQuery() { // @AGG removed callback StatementCallbackInterface statementI) {
```

Comment:
In this file I'm intentionally leaving the commented out code in place because this is the code that's ported from Apache Derby and I'm keeping the commented out code for easy reference in the near future.